onnxruntime/core/mlas/lib/mlasi.h

```diff
@@ -782,7 +782,7 @@ extern "C" {
 // value.
 //
 
-#define MLAS_DEFAULT_PREFERRED_BUFFER_ALIGNMENT     32
+#define MLAS_DEFAULT_PREFERRED_BUFFER_ALIGNMENT     64
```

Comment:
On CPUs with AVX-512, the default alignment for mlas is 32. So this PR will change it to 64. For whom only use the default CPU EP, this PR only has negative impacts.

Reply:
false. https://github.com/microsoft/onnxruntime/issues/13119

Reply:
@snnn, did you observe negative impact changing alignment from 32 to 64? I guess the impact shall be small (except potential a little more memory usage).

Reply:
No, I didn't.